apollo-android-support/src/main/java/com/apollographql/apollo/cache/normalized/sql/SqlNormalizedCache.java

```diff
@@ -66,18 +82,20 @@
     } else {
       Record oldRecord = optionalOldRecord.get();
       changedKeys = oldRecord.mergeWith(apolloRecord);
-      if (!changedKeys.isEmpty()) {
-        updateRecord(oldRecord.key(), recordAdapter().toJson(oldRecord.fields()));
-      }
+      updateRecord(oldRecord.key(), recordAdapter().toJson(oldRecord.fields()));
```

Comment:
Good catch, I was initially going to put in storing lastModified in this PR, but held off. Will revert this util it makes more sense.